pkg/types/main.go

```diff
@@ -153,12 +153,11 @@ func (pyroscopeCfg ReceptorPyroscopeCfg) Init() error {
 		DisableGCRuns:     pyroscopeCfg.DisableGCRuns,
 		HTTPHeaders:       pyroscopeCfg.HTTPHeaders,
 	})
-
 	if err != nil {
 		return err
-	} else {
-		return nil
 	}
+
+	return nil
```

Comment:
As @fincamd suggests let's simply return `err` as that catches both cases, where an error exists or doesn't.

```suggestion

    return err
```